src/state/experiment.js

```diff
@@ -0,0 +1,266 @@
+import { createSelector } from 'reselect';
+import { LOCATION_CHANGED, replace, push } from 'redux-little-router';
+import api from '../api';
+import { namedRoutes, makeUrl } from './routes';
+import { add as addErrors } from './errors';
+
+export const ANSWER_SIMILAR = 'Yes';
+export const ANSWER_MAYBE = 'Maybe';
+export const ANSWER_DIFFERENT = 'No';
+export const ANSWER_SKIP = 'Skip';
```

Comment:
Minor comment on consistency: Buttons are labeled Identical, Similar, Different, while the drop-down is Yes, Maybe, No

Reply:
I'll change them as soon as API will be ready.

Reply:
can be handled in https://github.com/src-d/code-annotation/issues/17

Reply:
Dropdown labels and buttons texts should not depend on the API but totally decoupled.

Reply:
what? sure they are decoupled. From my understanding, the idea of @carlosms is to reconsider constants values because they don't reflect what we are actually doing. Right now is the best time for it.

Reply:
I understand they're somehow coupled to the API since you mentioned it will be fixed when the API is ready.
https://github.com/src-d/code-annotation/pull/9#discussion_r162379976
If the dropdown labels shows the values sent/recieved from the API, it's wrong: there should be a mapping between the strings shown in the UI, and the values handled by the API, because the API could work with non human constants, and the UI should use user friendly wording.

Reply:
read code. they aren't coupled.

Reply:
So... since the API does not affect these labels, there is no reason because dropdown and buttons currently use different labels, there is?